plopfile.js

```diff
@@ -50,13 +50,11 @@ module.exports = function (plop) {
         const indexFile = `${plop.getDestBasePath()}/packages/components/src/index.ts`;
 
         if (fs.existsSync(indexFile)) {
-          const nameRegex = /default as ([^\s]*)/;
-          const sorted = fs
-            .readFileSync(indexFile, "utf8")
-            .split(";\n")
+          const lines = fs.readFileSync(indexFile, "utf8").split(";\n");
+          const sorted = lines
             .sort((a, b) => {
-              const aName = a.match(nameRegex)[1];
-              const bName = b.match(nameRegex)[1];
```

Comment:
Why [1] and not [0]? 🤔

Reply:
`match` returns [full match, ...capturing group(s)] right?

Reply:
You're probably right. Would explain why `export type { foo }` went to the bottom of the list.